test/internet/test-http-dns-fail.js

```diff
@@ -4,7 +4,6 @@
  * should trigger the error event after each attempt.
  */
 
-var common = require('../common');
```

Comment:
Could also change `var common = require('../common.js')` to just `require('../common.js');`...